pkg/data/managers/base.go

```diff
@@ -39,6 +39,12 @@ type BaseManager interface {
 	// The user would not see any counts beyond the scope
 	// but will see the total count beyond the filter
 	GetPageInfo(ctx context.Context, table string, page parameters.Page, scope, filter squirrel.Sqlizer) (pageInfo PageInfo, err error)
+	// GetPageInfoWithQuery is a more low-level version of `GetPageInfo` that allow to use an arbitrary
```

Comment:
My only question is if we should deprecate the `GetPageInfo`, I think you can add `Deprecated: GetPageInfo` to the beginning of the doc string or add `Deprecate: <msg>` as the last line of the doc string and it behaves nicely with godoc

Reply:
it was not my intent, I wanted to keep both, sometimes it's easier to use the previous version when the query is simple.